packages/inter-protocol/test/psm/gov-replace-committee.js

```diff
@@ -6,15 +6,13 @@
  *   (aka big hammer)
  */
 
-import { Fail } from '@endo/errors';
```

Comment:
`Fail` is available on the `assert` global:

```js
const { Fail } = assert;
```

you might want to add...
```js
/// <reference types="@agoric/vats/src/core/core-eval-env"/>
/// <reference types="@agoric/vats/src/core/types-ambient"/>
```

We should probably document this in https://docs.agoric.com/guides/coreeval/permissions.html#declaring-required-capabilities